src/domain/challenge/space/space.service.ts

```diff
@@ -399,6 +402,7 @@ export class SpaceService {
     if (args && args.IDs)
       spaces = await this.spaceRepository.find({
         where: { id: In(args.IDs) },
```

Comment:
you have to also include `options.where`, because the rest operator below will override everything